openai/cli.py

```diff
@@ -541,6 +541,7 @@ def signal_handler(sig, frame):
 
         events = openai.FineTune.stream_events(job_id)
         # TODO(rachel): Add a nifty spinner here.
+        spinner = itertools.cycle(['-', '\\', '|', '/']) # Spinner characters
```

Comment:
Thanks for the PR! A few points
1. This FineTune SDK [is being deprecated](https://platform.openai.com/docs/deprecations#:~:text=On%20August%2022nd%2C%202023%2C%20we,down%20on%20January%2004%2C%202024.) so it probably doesn't make sense to update this code
2. I don't think the spinner is being actually used anywhere? It's just defined